Components/Coroutines/Town/TrinityTownRun.cs

```diff
@@ -134,23 +145,24 @@ public static async Task<bool> IdentifyItems()
 
         public static async Task<bool> DoTownRun()
         {
-            if (!ZetaDia.IsInGame) return true;
+            // We're dead, wait till we're alive again...
+            if (!ZetaDia.IsInGame ||
+                ZetaDia.Me.IsDead)
+                return true;

```

Comment:
Add curly brackets when the statement is multi-line. Although the statement can be one liner here